### ISSUE TITLE
Error 422 response from github api

### ISSUE THREAD

**Issue by nathan815** [Neovici/github-actions-eslint-annotator#14]
I am seeing the following error when github-actions-eslint-annotator [runs on a pull request](https://github.com/seeus-dev/seeus-app/pull/29/checks?check_run_id=535041342).

```
yarn run v1.22.4
$ /home/runner/work/seeus-app/seeus-app/node_modules/.bin/github-actions-eslint-annotator
Error Error: Received status code 422
    at IncomingMessage.res.on (/home/runner/work/seeus-app/seeus-app/node_modules/@neovici/github-actions-eslint-annotator/index.js:35:20)
    at IncomingMessage.emit (events.js:203:15)
    at endReadableNT (_stream_readable.js:1145:12)
    at process._tickCallback (internal/process/next_tick.js:63:19)
{"message":"Invalid request.\n\n\"end_line\", \"start_line\" weren't supplied.","documentation_url":"https://developer.github.com/v3/checks/runs/#update-a-check-run"}
error Command failed with exit code 1.
```

It's causing the check to fail. Yet the annotations are still showing up, so I am not sure what is causing this.

Here is my workflow yaml:

```
name: CI
on:
  pull_request:
    branches:
      - '*'
  push:
    branches:
      - master
jobs:
  build:
    runs-on: ubuntu-latest
    steps:
      - name: Checkout
        uses: actions/checkout@v1

      - name: Use Node.js 10.x
        uses: actions/setup-node@v1
        with:
          node-version: 10.x

      - name: Install
        run: yarn install --frozen-lockfile

      - name: ESLint
        run: yarn lint

      - name: ESLint annotate
        run: yarn add @neovici/github-actions-eslint-annotator && yarn run github-actions-eslint-annotator
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
          PARTIAL_LINTING: true
```

**Comment by nathan815** [comment]
The error is that start_line and end_line aren't provided. Looking into the code, it looks like this could be caused by `m.line` not being populated when it processes the annotations in [index.js:119](https://github.com/Neovici/github-actions-eslint-annotator/blob/master/index.js#L119). Maybe it'd be good to filter out items from `annoList` where `m.line` is undefined/null before the map operation.

**Comment by nomego** [comment]
Hi @nathan815 and great that you've found this package useful.
Just a few days ago, we started exploring rather using the eslint problem matcher of setup-node, have you evaluated this route? If yes, what made you choose our package instead?

I made an example PR to one of our components at https://github.com/Neovici/cosmoz-utils/pull/50/files and it seems to report as well as our package so we're currently headed towards deprecating this package.
